src/app/views/pages/clientes/clientes.component.html

```diff
@@ -29,16 +29,33 @@ <h4 class="card-title">CLIENTES</h4>
                     <h6 class="card-title">Clientes</h6>
                     <div class="table-responsive">
                         <ngx-datatable class="bootstrap" [rows]="clientes" [loadingIndicator]="loadingIndicator" [columnMode]="ColumnMode.force" [footerHeight]="50" [limit]="10" rowHeight="auto">
-                            <ngx-datatable-column name="Id" prop="idCliente"></ngx-datatable-column>
+                            <ngx-datatable-column [width]="25" name="Id" prop="idCliente"></ngx-datatable-column>
                             <ngx-datatable-column name="Nombre" prop="nombre"></ngx-datatable-column>
                             <ngx-datatable-column name="Razon Social" prop="razonSocial"></ngx-datatable-column>
                             <ngx-datatable-column name="Contacto" prop="contacto"></ngx-datatable-column>
                             <ngx-datatable-column name="Telefono" prop="telefono"></ngx-datatable-column>
                             <ngx-datatable-column name="Mail" prop="mail"></ngx-datatable-column>
-                            <ngx-datatable-column name="Fecha de alta" prop="fechaAlta"></ngx-datatable-column>
-                            <ngx-datatable-column name="Fecha Actualizacion" prop="fechaActualizacion"></ngx-datatable-column>
+                            <ngx-datatable-column name="Fecha de alta" prop="fechaAlta">
```

Comment:
Muy bien implementada es fecha we, le vamos sacando el potencial a esa tabla Felicidades